lib/db/util.js

```diff
@@ -46,6 +47,14 @@ module.exports = {
     } else {
       return BOUNCE_SUB_TYPES.get(val) || 0
     }
-  }
+  },
 
+  createHash () {
+    const hash = crypto.createHash('sha256')
+    // Use ...rest operator and forEach when we're on node 6
```

Comment:
I thought we were on node 6 already?

Reply:
@rfk confirm?

Reply:
Yes, I believe we are comfortably on node 6; if this repo is still testing node 4 support, feel free to drop it

Reply:
Updated!